Sources/EdgeState.swift

```diff
@@ -62,6 +75,9 @@ class EdgeState {
         }
         // else keep consent pending until the consent preferences update event is received
 
+        // TODO - Always set state or only when properties are not empty ???
+        createSharedState(edgeProperties.toEventData(), nil)
```

Comment:
Always creates a shared state when EdgeState boots, regardless of location hint value. Another option is to only create the shared state if there is a valid location hint value, but I think creating an initial empty shared state on boot is better approach and more consistent with how other extensions are implemented.